node/process.ts

```diff
@@ -560,18 +560,6 @@ class Process extends EventEmitter {
     return 0o22;
   }
 
-  /** https://nodejs.org/api/process.html#processgetuid */
-  getuid(): number {
```

Comment:
how about keeping only the typing like `getuid?: () => number;` and call it like `getuid?.()` in testing? (I guess that reduces the type errors in test cases)

Note: DefinitelyTyped defines this as `getuid?: () => number;` https://github.com/DefinitelyTyped/DefinitelyTyped/blob/35b81ce6b7865843f9c2e49861c470621134ce3b/types/node/process.d.ts#L689